src/main/java/com/google/api/codegen/config/PackagingConfig.java

```diff
@@ -28,12 +30,18 @@
 import java.nio.file.Paths;
 import java.util.List;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import javax.annotation.Nullable;
 import org.yaml.snakeyaml.Yaml;
 
 /** Represents the packaging config necessary to run package generation for an API. */
 @AutoValue
 public abstract class PackagingConfig {
+  private static final Pattern COMMON_PKG_PATTERN =
+      Pattern.compile("(?i)(?<org>(\\w+\\.){0,2})(?<name>\\w+)\\.(?<ver>\\w*\\d+[\\w\\d]*)(\\.|$)");
```

Comment:
This doesn't seem to work for grafeas Java gapic?
https://github.com/googleapis/googleapis/blob/master/grafeas/v1/grafeas_gapic.yaml

Reply:
Talked offline: different issue.

Reply:
As we discussed offline, this should not matter.